### PR TITLE
Adding to markdown

### PR DESCRIPTION
**Pull request by botany-demo**
please take a look at this

### CODE REVIEW

File: bugbash.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
 # Bug Bash File For Activity
 Who goes first?
+
+## I Can Make Next PR
+- hamburgers
```
Comment (botany-demo) [inline bugbash.md]: changed order of food

Comment (botany-dummy) [reply bugbash.md]: nice 👍 - we should add one more if you can think of any

Comment (botany-demo) [reply bugbash.md]: ok yeah that makes sense

File: bugbash.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
...
+- ice cream
```
Comment (botany-dummy) [inline bugbash.md]: good idea

**Pull request merged**